### PR TITLE
Community quick wins oct 23

### PR DESCRIPTION
**Pull request by oetherington**
This PR makes 2 small changes:

- Quick takes tagged with community are no longer featured in the frontpage quick takes section
- Comments posts tagged with community are excluded from the frontpage popular comments section

Adding the ability to easily edit tags on quick takes will be a separate PR.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205678813147791) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jpaddison3** [comment]
Could we have a terms item that allows quick takes to have community if the author has un-collapsed the community section? This is a pretty big change for community quick takes authors

**Comment by agnesstenlund** [comment]
Yup I'm changing my mind on this too, we shouldn't just hide it. I'll explore a few versions of a filtering UI like the one on All posts, will share in a bit

**Comment by agnesstenlund** [comment]
Let's do this:

- [ ] Add checkbox on the left of "View more"
Screenshot 2023-10-12 at 18 24 41

- [ ] Show tooltip when hovering "Show community" that reads `Quick takes tagged "Community"`

- [ ] Default for toggle should be inherited 1. from the "Hide community section from the frontpage" in account settings, and if that's not toggled, 2. From the toggle state of the community section the time we ship this (shouldn't update continously)

If this is really tricky 👆 I'd also be ok just hiding by default, since it's easy to spot and toggle on.

**Comment by oetherington** [comment]
@agnesstenlund Done!

Screenshot 2023-10-13 at 14 27 30

**Comment by agnesstenlund** [comment]
looks great! These are looking a tiny bit misaligned–the "View more" a few pixels further down. 
Screenshot 2023-10-16 at 10 50 05

Otherwise good to go (unless we to do the [section thing] 😬 (https://cea-core.slack.com/archives/C03G0AQ925P/p1697194435488859?thread_ts=1697022406.431089&cid=C03G0AQ925P))

**Comment by oetherington** [comment]
@agnesstenlund I just added a commit to fix the alignment of the text. I _think_ it's correct now, but I'm actually having a really hard time trying to tell if it's exactly right or not :sweat_smile:.

**Comment by agnesstenlund** [comment]
This is definitely close enough 👍 

Screenshot 2023-10-16 at 16 27 00

File: packages/lesswrong/components/common/ExpandableSection.tsx
Comment (s-cheng) [inline packages/lesswrong/components/common/ExpandableSection.tsx:92]: These things get a bit squished on small screens, and then the checkbox is hidden entirely on the smallest screens.
Screen Shot 2023-10-16 at 3 53 41 PM

Probably the "Show community" checkbox is important to display on all screen sizes, but the "View more" is ok to lose on small screens. Maybe we just make the checkbox wrap onto a second line.

Comment (s-cheng) [reply packages/lesswrong/components/common/ExpandableSection.tsx:92]: Also unfortunately things are not quite aligned in Safari:

Screen Shot 2023-10-16 at 4 19 06 PM

File: packages/lesswrong/components/quickTakes/QuickTakesSection.tsx
Comment (s-cheng) [inline packages/lesswrong/components/quickTakes/QuickTakesSection.tsx:52]: I think this interaction is a bit weird. The "Show community" quick takes checkbox depends on the previous time I expanded/collapsed the "Community posts" section, until I explicitly check or uncheck the "Show community" checkbox. So like, if I expand "Community posts" now, upon refresh, the checkbox is checked.

Maybe the thing we want is to set the cookie the first time the user loads this section. That way it won't sometimes change on them unexpectedly.

**Review by s-cheng — APPROVED** [review]
ASTLAC

I think we also still have the buggy interaction between the logged out user cache and the expandable sections. Didn't seem too bad when I was testing, so prob ok to worry about later.

File: packages/lesswrong/components/common/ExpandableSection.tsx
Comment (agnesstenlund) [reply packages/lesswrong/components/common/ExpandableSection.tsx:92]: Thanks so much for catching these 😮 

> Probably the "Show community" checkbox is important to display on all screen sizes, but the "View more" is ok to lose on small screens. Maybe we just make the checkbox wrap onto a second line.

I would probably guess the opposite since the community checkbox is more of a one-off thing to do, whereas you may want to view more more often (not strongly held though). So I'd probably hide the checkbox instead

Comment (s-cheng) [reply packages/lesswrong/components/common/ExpandableSection.tsx:92]: Yeah I'm not really sure which is best. I was thinking that there is still a "Load more" button so you can still see more quick takes that way on mobile. But if you are just a mobile user, you might not realize that community quick takes are being filtered out at all without the checkbox.

**Comment by oetherington** [comment]
@s-cheng I couldn't reproduce the alignment bug in Safari, but I made a change that I _think_ should fix it. If you have a spare minute could you quickly test if it's fixed before I merge? Thanks!

**Comment by s-cheng** [comment]
@oetherington Yup it does look better now, although the arrows are still slightly off...

Screen Shot 2023-10-19 at 6 31 37 PM

**Comment by oetherington** [comment]
Damn - any chance you could have a quick go at fixing it as I still can't repro?

My guess is that it could be fixed by `adding height: 0` and `margin-bottom: 20` to `.ExpandableSection-title`? Or maybe just some `justify-content` or `align-items` value?

**Comment by s-cheng** [comment]
@oetherington Sure thing - I made a quick change and I think this is better overall, although it's very slightly off center on chrome

**Comment by oetherington** [comment]
@s-cheng Amazing - thanks! I just moved the toggle up 1px to make it nicer in Chrome and Firefox. If it's 1px too high in Safari now then people should stop using Safari (still looks fine for me though - I think it must be a specific version that's problematic).

**Pull request merged**
